File: basic_python/HELLOJAVA/src/day07/MyInsert.java
package day07;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.Statement;

public class MyInsert {
	
	public static void main(String[] args) throws Exception {
		

		String driver = "oracle.jdbc.driver.OracleDriver";
		String url = "jdbc:oracle:thin:@localhost:1521:xe"; 
		String sql = "insert into sample(col01, col02, col03) values ('6','6','6')";
		
		Class.forName(driver);
		Connection conn = DriverManager.getConnection(url, "python", "python");
		Statement stmt = conn.createStatement();
		int cnt = stmt.executeUpdate(sql);
		
		System.out.println("cnt:"+cnt);
		
		stmt.close();
		conn.close();
		

	}
	
	
}

File: basic_python/HELLOPYTHON/day08/myinsert.py
import cx_Oracle

#연결에 필요한 기본 정보 (유저, 비밀번호, 데이터베이스 서버 주소)
#('유저/비밀번호@데이터베이스서버주소')
connection = cx_Oracle.connect('python','python','localhost:1521/xe')
cursor = connection.cursor()

sql_str = "insert into sample(col01, col02, col03) values(:1,:2,:3)"
cnt = cursor.execute(sql_str,('77','77','77'))

print(cnt)

connection.commit()



cursor.close()
connection.close()